checkoff

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import subprocess
+import sys
+
+from ocflib.account.search import user_exists
+
+
+# Format of JSON file should be:
```

Comment:
We would just need to manually generate a JSON file like so:

```
{
    "roster": ["filled", "with", "initial", "roster"],
    "users": {},
    "labs": {}
}
```

and we can start using the script